x/qgb/client/rest/tx.go

```diff
@@ -0,0 +1,3 @@
+package rest
+
+// TODO add ValSetConfirm REST methods
```

Comment:
grpc is already up and running, so I'm not sure adding a rest endpoint really gives us anything, but wdyt?

Reply:
True that. I'll leave it for later if needed